demo/cache/cache.html

```diff
@@ -1,61 +1,93 @@
-<!-- Page Content -->
+<uib-accordion close-others="oneAtATime"> <!-- FDP caching config -->
+<div uib-accordion-group class="panel-info">
+	<uib-accordion-heading> <span
+		class="glyphicon glyphicon-cog"></span> FDP cache configuration <i
+		class="pull-right glyphicon"
+		ng-class="{'glyphicon-chevron-down': status.open, 'glyphicon-chevron-right': !status.open}"></i>
+	</uib-accordion-heading>
+	<form>
+		<div class="form-group row" ng-repeat="fdp in fairDataPoints">
+			<div class="col-lg-4">
+				<div class="input-group">
+					<span class="input-group-addon">String</span> <input type="text"
+						class="form-control" aria-label="..." placeholder="Enter FDP name"
+						ng-model="fdp.name">
+				</div>
+			</div>
+			<div class="col-lg-6">
+				<div class="input-group">
+					<span class="input-group-addon">Url</span> <input type="text"
+						class="form-control" aria-label="..." placeholder="Enter FDP URL"
+						ng-model="fdp.url">
+				</div>
+			</div>
+			<div class="col-lg-2">
+				<div class="input-group">
+					<div></div>
```

Comment:
Is this empty div here for a reason?

Reply:
Not really, I think I used to add empty line